gis/pgrouting/Portfile

```diff
@@ -18,7 +18,7 @@ license             LGPL-2
 homepage            http://www.pgrouting.org/
 
 github.setup        pgRouting pgrouting 3.6.2 v
-revision            0
+revision            1
```

Comment:
If you are only fixing a build problem, then the revision should not be increased. See the documentation of the revision keyword at https://guide.macports.org/#reference.keywords.